test/src/oopif.spec.ts

```diff
@@ -418,7 +418,7 @@ describe('OOPIF', function () {
       return target.url().endsWith('dynamic-oopif.html');
     });
     await target.page();
-    browser1.disconnect();
+    await browser1.disconnect();
```

Comment:
can we still keep the old way of disconnecting to make sure it's not breaking?

Reply:
btw, now that I think of it, did we verify that it does not work when the disconnect happens asynchronously?

Reply:
Can expect the firefox to finish processing session.end before starting to handle session.new?

Reply:
let me ask

Reply:
I mean there should be a mutex on the shared resource that is a session, right?

Reply:
But we cannot close connection before the `session.end` command is processed. Meaning `disconnect` will not disconnect immediately, but rather eventually, which is confusing.